sympy/parsing/mathematica.py

```diff
@@ -8,126 +9,218 @@ def mathematica(s):
     return sympify(parse(s))
 
 
-class _Parse(object):
-    '''An instance of this class converts basic Mathematica expression to
-    Python.'''
+class parse(object):
```

Comment:
I usually prefer to have unique class names in the project, to avoid name collisions. But it's OK.

Reply:
So, 'mathematcia_parse' could be better from the view point of name collisions.
But in this case, I also think it's OK.